m4/pdns_enable_reproducible.m4

```diff
@@ -12,7 +12,7 @@ AC_DEFUN([PDNS_ENABLE_REPRODUCIBLE], [
     AC_DEFINE([REPRODUCIBLE], [1], [Define to 1 for reproducible builds])
   ],[
     build_user=$(id -u -n)
-    if [ test x"$host_os" = "xSunOS" ]; then
+    if [ test x"$host_os" = "xSunOS" -o x"$host_os" = "xsolaris2.11" ]; then
```

Comment:
You could use AC_REQUIRE([PDNS_CHECK_OS])

Btw, I'm wondering when $host_os is ever SunOS? The systems I've seen always have something like solaris*

Otherwise LGTM.